packages/features/bookings/Booker/Booker.tsx

```diff
@@ -144,7 +144,7 @@ const BookerComponent = ({
               )}>
               <BookerSection
                 area="meta"
-                className="max-w-screen flex h-full w-full flex-col md:w-[var(--booker-meta-width)]">
```

Comment:
This incorrectly limited the meta side column height if contents was smaller than screen.